corehq/apps/locations/templates/locations/downgrade_locations.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'settings/base_template.html' %}
+{% load i18n %}
+{% load hq_shared_tags %}
+
+{% block main_column %}
+<div class="span6">
+    <h2>{% trans "Organizations" %}</h2>
+
+    <p>
+        {% blocktrans %}
+        Your project no longer has access to the Organization feature. To
+        access this feature, you must subscribe to the Standard plan or
+        higher. Until you change your subscription or discontinue your use of
+        Organization featuress, you will not be able to add or remove mobile workers
+        from groups. Fix this using one of the options below:
+        {% endblocktrans %}
+    </p>
+
+    <p>
+        <a class="btn btn-primary" href="[button links to Current Subscription page]">
+            {% trans "Change your subscription" %}
```

Comment:
Might change this to reference which plan again; the help text above is pretty long to expect people to read. "Upgrade your subscription to Standard"?